src/locales/fr.yml

```
@@ -2,26 +2,28 @@ commands:
   novelai:
     description: Générer des images sur Novel AI
     usage: |-
-      Entrez « /novelai » avec une description textuelle (prompt) ou des balises. Par ex. a girl in the forest, blonde hair, red eyes, white dress.
+      Entrez « /novelai » avec les descriptions textuelles (anglais : prompt) ou des balises. Par ex. a girl in the forest, blonde hair, red eyes, white dress.
       Vous pouvez utiliser des balises séparées par des virgules, comme sur Danbooru.
       Donnez-lui une étoile : https://github.com/koishijs/novelai-bot
 
     options:
       enhance: Mode d'amélioration de l'image
       model: Définir le modèle pour génération
       resolution: Définir la taille de l'image
+      override: Remplacer les descriptions textuelles de base
       sampler: Définir l'échantillonneur
-      seed: Définir une graine aléatoire
+      seed: Définir la graine aléatoire
       steps: Définir les étapes de l'itération
       scale: Définir CFG Scale
       strength: Définir l'intensité de débruitage
       noise: Définir l'intensité du bruit
+      undesired: Définir les descriptions textuelles négatives
 
     messages:
-      expect-prompt: Attendrez-vous une description textuelle valide.
+      expect-prompt: Attendrez-vous les descriptions textuelles valides.
       expect-image: Attendrez-vous une image.
-      latin-only: L'description textuelle n'est pas valide, vous ne pouvez utiliser que des mots anglais.
-      forbidden-word: L'description textuelle contient des mots prohibés.
+      latin-only: Les descriptions textuelles ne sont pas valides, vous ne pouvez utiliser que des mots anglais.
+      forbidden-word: Les descriptions textuelles contient des mots prohibés.
```

```suggestion
      forbidden-word: Les descriptions textuelles contiennent des mots prohibés.
```

verb should be plural too.